vars/runATH.groovy

```diff
@@ -12,7 +12,7 @@ def call(Map params = [:]) {
     def jdks = params.get('jdks', [8])
     def athContainerImageTag = params.get("athImage", "jenkins/ath");
     def configFile = params.get("configFile", null)
-    def defaultJavaOptions = params.get('javaOptions', ["-Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn"])
+    def defaultJavaOptions = params.get('javaOptions', [])
```

Comment:
```suggestion
    def defaultJavaOptions = params.get('javaOptions', ['--no-transfer-progress'])
```

Reply:
reverted in https://github.com/jenkins-infra/pipeline-library/pull/244